cardano-cli/src/Cardano/CLI/EraBased/Run/Transaction.hs

```diff
@@ -1354,11 +1354,11 @@ createTxMintValue era (val, scriptWitnesses) =
         era
  where
   gatherMintingWitnesses
-    :: [ScriptWitness WitCtxMint era]
+    :: [(Maybe PolicyId, ScriptWitness WitCtxMint era)]
     -> [(PolicyId, ScriptWitness WitCtxMint era)]
   gatherMintingWitnesses [] = []
-  gatherMintingWitnesses (sWit : rest) =
-    case scriptWitnessPolicyId sWit of
+  gatherMintingWitnesses ((pid'm, sWit) : rest) =
+    case scriptWitnessPolicyId pid'm sWit of
```

Comment:
```suggestion
  gatherMintingWitnesses ((mPid, sWit) : rest) =
    case scriptWitnessPolicyId mPid sWit of
```

Rationale: having a quote in the middle of a variable name is surprising

Reply:
I picked up this style in Standard Chartered. No issue, I will adhere to suggested style.

Reply:
All updated, squashed and formatted.